src/constants/currency.ts

```diff
@@ -52,6 +52,7 @@ export const addressToCurrencyKeyMap = {
     cx2e6d0fc0eca04965d06038c8406093337f085fcf: 'CFT',
     cx369a5f4ce4f4648dfc96ba0c8229be0693b4eca2: 'METX',
     cx3a36ea1f6b9aa3d2dd9cb68e8987bcc3aabaaa88: 'IUSDT',
+    cx785d504f44b5d2c8dac04c5a1ecd75f18ee57d16: 'FIN',
```

Comment:
The `addressToCurrecyKeyMap` will keep three tokens that the BalancedDAO fund has. sICX, bnUSD, BALN. You don't need to add `FIN` token here.